rd_ui/app/views/query.html

```diff
@@ -192,6 +192,16 @@ <h4 class="modal-title">Query Archive</h4>
             </div>
             <div class="alert alert-danger" ng-show="queryResult.getError()">Error running query: <strong>{{queryResult.getError()}}</strong></div>
 
+            <div class="row log-container" ng-show="showLog">
```

Comment:
I checked and it basically hides without taking any height, so the margin doesn't affect anything.